apps/documents/templatetags/react_paragraphs.py

```diff
@@ -25,7 +27,7 @@ def react_paragraphs(context, doc, module):
     context = {
         'document': document,
```

Comment:
We discussed this feature earlier and you were quire eager to also use `json.dumps` for the document value. Did you have any second thoughts on that?